ddtrace/contrib/google_generativeai/__init__.py

```diff
@@ -0,0 +1,92 @@
+"""
+The Gemini integration instruments the Google Gemini Python API to traces for requests made to Google models.
+
+All traces submitted from the Gemini integration are tagged by:
+
+- ``service``, ``env``, ``version``: see the `Unified Service Tagging docs <https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging>`_.
+- ``genai.request.model``: Google model used in the request.
```

Comment:
The `genai` prefix is supposed to be specific to this integration right? Should we change it to `google_genai.request.model` to clarify? It's verbose, but imo clarity > verbosity

Reply:
I agree with this one. Feels like genai would easily be mistaken for the general term rather than the gemini case.